### PR TITLE
Content item update

### PR DESCRIPTION
**Pull request by daanishnasir**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/collab-ui/collab-ui-react/blob/master/CONTRIBUTING.md#commit-message-format
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
- [x] Bugfix
- [ ] Feature
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

Add import of ContentItem to example
Add file ellipsis for text overflow to file subtitle and chat title. 
omit gifIcon prop to IconContent
Borders are consisted with IconContent. 
Add position relative, actionNode Icons were moving with IconContent 

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/lib/ContentItem/index.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import ChatContentItem from '@collab-ui/react/ContentItem/ChatContent';
 import FileContentItem from '@collab-ui/react/ContentItem/FileContent';
 import IconContent from '@collab-ui/react/ContentItem/IconContent';
 
+import { omit } from 'lodash';
```
Comment (bfbiggs) [inline src/lib/ContentItem/index.js]: Not a blocker but could remove the space before this line.

Comment (daanishnasir) [reply src/lib/ContentItem/index.js]: removed

Comment (daanishnasir) [reply src/lib/ContentItem/index.js]: It's removed on PR.

Comment (pauljeter) [reply src/lib/ContentItem/index.js]: Already merged.

**Pull request merged**